hello_wifi/mix.exs

```diff
@@ -70,14 +75,11 @@ defmodule HelloWifi.Mixfile do
      "deps.loadpaths":  ["deps.loadpaths", "nerves.loadpaths"]]
   end
 
+  def kernel_modules("rpi0"), do: ["brcmfmac"]
```

Comment:
I think we should consolidate it with rpi3, which has also builtin support for WiFi with the same chip. From my point of view, we should support standard hardware, if there is support for the user land part. Therefore, the bluetooth drivers should not be part of the regular build but stay modularized. Controlling bluetooth with Nerves is still in statu nascendi